reconcile/utils/state.py

```diff
@@ -4,14 +4,43 @@
 from typing import Any, Iterable, Mapping, Optional
 
 from botocore.errorfactory import ClientError
+from jinja2 import Template
+from reconcile.utils import gql
 
 from reconcile.utils.aws_api import AWSApi
+from reconcile.utils.secret_reader import SecretReader
 
 
 class StateInaccessibleException(Exception):
     pass
 
 
+STATE_ACCOUNT_QUERY = """
+{
+  accounts: awsaccounts_v1 (name: "{{ name }}")
+  {
+    name
+    resourcesDefaultRegion
+    automationToken {
+      path
+      field
+      version
+      format
+    }
+  }
+}
+"""
```

Comment:
This query would be a great new example for the query class generation. It contains an input variable and we dont have an example for that yet. 

If its ok for you, I will create a follow-up PR to convert this.

Reply:
i want to learn about this from the master personally so wdyt about peering on this?

Reply:
thanks for the session today. it turned out quite easy to implement but would require a lot of code to be touched (AWSApi, State, SecretReader are very widely used central pieces of code). i'm going to postpone that to another PR if that is ok with you.